features/admin.tenants.v1/pages/edit-tenant-page.scss

```diff
@@ -45,4 +45,39 @@
             }
         }
     }
+
+    .console-url-copy-field {
+        .MuiInputBase-root {
+            min-width: 400px;
+
+            &.Mui-readOnly {
+                cursor: default;
+                background-color: #e9ecef!important;
```

Comment:
shall we use a SCSS variable for the color?

Reply:
This is a temporarily thing that needs to be fixed with Oxygen UI.
We should not encourage to re-use a variable. So, lets link and issue here instead.